config/watches.yaml

```diff
@@ -3,6 +3,8 @@
   version: v1alpha1
   kind: Nginx
   chart: /var/helm-charts/ingress-nginx
+  overrideValues:
```

Comment:
I would recommend changing the Nginx CR directly, the file located under:
https://github.com/kubesphere/kubesphere/blob/master/config/gateway/templates/nginx-ingress.yaml
The  OverrideValues are used when overriding values with environment variables.